tests/services/components/test_internal_notes_component.py

```diff
@@ -0,0 +1,105 @@
+# -*- coding: utf-8 -*-
+#
+# Copyright (C) 2021 CERN.
+#
+# Invenio-RDM-Records is free software; you can redistribute it and/or modify
+# it under the terms of the MIT License; see LICENSE file for more details.
+
+"""Tests for the service Metadata component."""
```

Comment:
I don't know if here or in [invenio-records-resources](https://github.com/inveniosoftware/invenio-records-resources/pull/599/files), but we need to see tests for searching text that is in the `internal_notes` field, without specifying the field name in the search query.

Reply:
done, in https://github.com/inveniosoftware/invenio-rdm-records/pull/1858/files#diff-fa2dc1cd0a4a4556f57462d4044d5a57208aec2455d7a237e84686641a38518dR951